themes/classic/templates/catalog/_partials/category-header.tpl

```diff
@@ -30,7 +30,7 @@
                 {if $category.description}
                     <div id="category-description" class="text-muted">{$category.description nofilter}</div>
                 {/if}
-                {if $category.image.large.url}
+                {if !empty($category.image.large.url)}
```

Comment:
Should there me a "no picture" fallback like for product?

Reply:
Since it's a category, I don't think there is :thinking: